examples/kubernetes-in-docker/demo_bootstrap.env

```diff
@@ -0,0 +1,31 @@
+. utils.sh
+
+# Basic demo config
+export CONJUR_NAMESPACE_NAME=conjur-oss
```

Comment:
So this demo requires you to have already ran the helm installation, where the instructions use `CONJUR_NAMESPACE`. Also, the values here may not match what the users set during their helm installation. For example, I deployed to the `default` namespace, so `conjur-oss` doesn't exist.

I think a better experience would be to remove the export statements here for variables the user has already set during their installation (CONJUR_ACCOUNT, NAMESPACE, etc.), otherwise we override them and the user might have to re-set them anyways. Besides, if the variables aren't set, the scripts throw an error anyways. We could even move the `check_env_var` check to this script.

Reply:
I see that the README.md suggests customizing this file, so this may be a moot point, but I'm not sure it should have any default values that can override an existing environment.

Reply:
What I can do to make this cleaner is to have the users continue to use the same environment variable names as they would use for Conjur OSS Helm chart... and in situations where the conjurdemos/kubernetes-conjur-demo scripts use a different naming convention, map the settings from the Helm chart names to the conjurdemos/kubernetes-conjur-demo script names.